cmd/boostd/run.go

```diff
@@ -43,7 +46,8 @@ var runCmd = &cli.Command{
 			}()
 		}
 
-		fullnodeApi, ncloser, err := lcli.GetFullNodeAPIV1(cctx)
+		subCh := gateway.NewEthSubHandler()
```

Comment:
Is there any performance implication of subscribing to these events? Wondering if we should check if fvm deals are enabled in config before subscribing.

Reply:
I don't think there is any performance implication, I think this is just creating the handler in memory, but the actual subscription happens elsewhere.